src/http_execution.jl

```diff
@@ -65,16 +65,21 @@ julia> query_string = \"\"\"
 julia> GraphQLClient.execute(client, query_string, operation_name="getCountries")
 ```
 """
+
+function CleanDict(pairs::Pair...)
+    return Dict(p for p in pairs if !isnothing(p[2]))
+end
+
 execute(query::AbstractString, output_type::Type{T}=Any; kwargs...) where T = execute(global_graphql_client(), query, output_type; kwargs...)
-function execute(client::Client, query::AbstractString, output_type::Type{T}=Any; variables=Dict(), operation_name="", kwargs...) where T
-    return execute(client, Dict("query" => query, "variables" => variables, "operationName" => operation_name), output_type; kwargs...)
+function execute(client::Client, query::AbstractString, output_type::Type{T}=Any; variables=Dict(), operation_name=nothing, kwargs...) where T
+    return execute(client, CleanDict("query" => query, "variables" => variables, "operationName" => operation_name), output_type; kwargs...)
```

Comment:
I don't think `CleanDict` is necessary here. `operation_name` being `nothing` is the same as it not being present

Reply:
You're right, null works just fine too!